tests/language-feature-scripts/lazy-loading-value-holder-initializer-removes-reference.phpt

```diff
@@ -0,0 +1,33 @@
+--TEST--
+Verifies that lazy loading value holder cannot be turned to a reference
+?>
+--FILE--
+<?php
+
+require_once __DIR__ . '/init.php';
+
+class Proxied
+{
+    public function compress($compression_type, $file_ext = null)
+    {
+        echo $compression_type;
+    }
+}
+
+$dummy = new stdClass();
+$factory = new \ProxyManager\Factory\LazyLoadingValueHolderFactory($configuration);
+
+$proxy = $factory
+    ->createProxy(Proxied::class, function (& $wrapped, $proxy, $method, array $parameters, & $initializer) use ($dummy) {
+        $initializer = null;
+        $wrapped     = new Proxied();
+        $dummy->reference = & $wrapped;
+    });
+
+$proxy->compress('Lazy ');
+$dummy->reference = 123;
```

Comment:
This is misuse of a reference... Not much I can do against it: that's a bug in the consumer code.

Reply:
```php
$proxy->compress('Lazy ');
$foo = 123;
$dummy->reference = & $foo;
$proxy->compress('Loaded!')
```

This is how it SHOULD be done.